openshift/kustomize/tekton/base/tasks/build-component.yaml

```diff
@@ -72,6 +72,9 @@ spec:
       script: |
         #!/usr/bin/env bash
 
+        export TMPDIR="/data/tmp/buildah"
```

Comment:
The docker contain was running out of space when building the API.  This places the build on the mapped volume instead.